### ISSUE TITLE
Pull and Deploy, no building (Feature Request)

### ISSUE THREAD

**Issue by soofstad** [equinor/radix-platform#176]
The CI part of Radix is somewhat limited(as it's outside the scope of Radix to be a fully fledged CI as I understand it), but our team would still like to benefit from the deployment capabilities of Radix.

A simple way to alleviate this would be to allow pulling of external images instead of building them in the CI/pipeline.

Proposal;
Allow omitting the "src" key in the radixconfig.yaml file for a component, and instead use an "image" key. 

As a first step, this could only pull publicly available images (from any container image repository that does not require authentication).

An example of an API that uses a Redis cache could look something like this;
```yaml
apiVersion: radix.equinor.com/v1
kind: RadixApplication
metadata:
  name: myAPI
spec:
  environments:
    - name: dev
      build:
        from: master
  components:
    - name: api
      src: ./api
      ports:
        - name: http
          port: 8080
      public: true
    - name: redis
        image: redis:5.0-alpine
```
The current workaround for this is adding a new component which has a simple Dockerfile like so;
```yaml
FROM redis:5.0-alpine
```

One could build upon this to also skip building your custom images, and use another tool for the CI part. As of now, we can not use the Radix CI to spin up my component, run some tests against a database, or deploy a schema crawler to document a DB's relational structure. 

For this kind of workflow to work, we would need to be able to trigger Radix deployments from an API call or a webhook from the external CI-tool. 
Alternatively you could have Radix periodically query a image repository for any new images, ether "latest", or matching a regex/glob tag filter, inspired by [Flux's automatic image deployment](https://github.com/fluxcd/flux/blob/master/site/annotations-tutorial.md#driving-flux)

**Comment by emirgens** [comment]
Thanks for reporting the request, it has been added to our product backlog and will be evaluated and if applicable prioritised and added to the queue.

Jira ref.: https://equinor.atlassian.net/browse/RA-510

**Comment by keaaa** [comment]
Really like these ideas. As you propose we will split this into two stories:
1. Being able to reference a (public) image instead of src/Dockerfile in radixconfig.yaml
2. Run a spike on using another CI and then deploy into radix. 

Because of vacation, it will take some time before we can start on these stories.

**Comment by emirgens** [comment]
The use of an public image is now a feature implemented in the Radix platform.

**Comment by emirgens** [comment]
This is implemented, closing this issue
